app/views/group.scala.html

```diff
@@ -14,47 +14,74 @@
 
 
-@this(form: FormWithCSRF)
+@this(
+		govuk_wrapper: govuk_wrapper,
+		formWithCSRF: FormWithCSRF,
+		govukBackLink: GovukBackLink,
+		govukErrorSummary: GovukErrorSummary,
+		govukRadios: GovukRadios,
+		govukButton: GovukButton
+)
 
 @(requestObject: RequestObject, groupScheme: Option[String], groupSchemeForm: Form[models.RS_groupScheme])(implicit request: Request[AnyRef], flash: Flash, messages: Messages, ersUtil: ERSUtil, appConfig: ApplicationConfig)
-@* Is a group scheme template *@
+
 @schemeId = @{requestObject.getSchemeId}
-@import views.html.templates._
-@import uk.gov.hmrc.play.views.html.helpers
 
-@ers_main(title = ersUtil.getPageElement(schemeId, "ers", "is_group_scheme.page_title") + " " + Messages("ers.title.postfix")) {
+@govuk_wrapper(title = ersUtil.getPageElement(schemeId, "ers", "is_group_scheme.page_title") + " " + Messages("ers.title.postfix")) {
 
 	@reference(requestObject.getPageTitle)
-	<p class="font-xsmall"><a href="@routes.SchemeOrganiserController.schemeOrganiserPage()" class="link-back">@Html(Messages("ers.back"))</a></p>
+
+	@govukBackLink(BackLink(
+		href = routes.SchemeOrganiserController.schemeOrganiserPage.url,
+	))
+
 	@if(groupSchemeForm.hasErrors) {
-		@error_overview(groupSchemeForm, Some(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_ACTIVITY, "err.message")))
+		@govukErrorSummary(ErrorSummary(
+			title = Text(messages("validation.summary.heading")),
+			errorList = groupSchemeForm.errors.asTextErrorLinks
+		))
 	}
-	@form(action = routes.GroupSchemeController.groupSchemeSelected(schemeId)) {
-		<div class="form-group @if(groupSchemeForm.hasErrors) {form-field--error}">
-			<fieldset class="inline">
-				<legend id="groupSchemeLbl">
-					@header(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_ACTIVITY, "question"))
-				</legend>
-				<div class="form-hint">
-					<span>@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_ACTIVITY, "intro_para"))</span>
-				</div>
-				@if(groupSchemeForm.hasErrors){<span class="error-notification">@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_ACTIVITY, "err.message"))</span>}
-				<div class="multiple-choice">
-					<input id="yes" type="radio" name="groupScheme" value="1" @(if(groupScheme.get == ersUtil.OPTION_YES) "checked" else "")>
-					<label for="yes" class="@(if(groupScheme contains ersUtil.OPTION_YES) "selected")">@Html(Messages("ers_group_scheme.yes"))</label>
-				</div>
-				<div class="multiple-choice">
-					<input id="no" type="radio" name="groupScheme" value="2" @(if(groupScheme.get == ersUtil.OPTION_NO) "checked" else "")>
-					<label for="no" class="@(if(groupScheme contains ersUtil.OPTION_NO) "selected")">@Html(Messages("ers_group_scheme.no"))</label>
-				</div>
-			</fieldset>
-		</div>
-		<input role="button" id="continue" class="button" type="submit" value="@Html(Messages("ers.continue"))">
+
+	@formWithCSRF(action = routes.GroupSchemeController.groupSchemeSelected(schemeId)) {
+
+		@govukRadios(Radios(
```

Comment:
this one seems to be missing the errorMessage parameter, so it only displays the error summary
![Uploading Screenshot 2021-05-11 at 16.51.06.png…]()